firmware/application/apps/ui_level.cpp

```diff
@@ -184,6 +184,8 @@ void LevelView::on_statistics_update(const ChannelStatistics& statistics) {
     rssi_graph.add_values(rssi.get_min(), rssi.get_avg(), rssi.get_max(), statistics.max_db);
 
     // refresh db
+    rssi.set_db(statistics.max_db);
```

Comment:
Shouldn't this be done in the  "if (last_max_db != statistics.max_db) {" following block ?